doc/tutorial.txt

```diff
@@ -422,6 +422,39 @@ following sequence:
 	"unlock" : "mutexA"
 }
 
+* barrier : String. Used as at least a pair where the name must match.
```

Comment:
Actually, the way I implemented it I think if there's only one reference you'll just emit a signal and carry on. I will test that it actually happens that way...